docs/getting_started.rst

```diff
@@ -0,0 +1,131 @@
+    Qiskit, i.e:
+
+    .. code:: sh
+
+        pip install qiskit[nature]
+
+    It is worth pointing out that if you're a zsh user (which is the default shell on newer
+    versions of macOS), you'll need to put ``qiskit[nature]`` in quotes:
+
+    .. code:: sh
+
+        pip install 'qiskit[nature]'
+
+
+.. tabbed:: Install from source
+
+   Installing Qiskit Nature from source allows you to access the most recently
+   updated version under development instead of using the version in the Python Package
+   Index (PyPI) repository. This will give you the ability to inspect and extend
+   the latest version of the Qiskit Nature code more efficiently.
+
+   Since Qiskit Nature depends on Qiskit, and its latest changes may require new or changed
+   feature from Qiskit, you should first `Install from source` Qiskit by following the instructions
+   here `Qiskit Getting Started <https://qiskit.org/documentation/getting_started.html>`__
+
+   .. raw:: html
+
+      <h2>Installing Qiskit Nature from Source</h2>
+
+   Using the same development environment that you installed Qiskit in you are ready to install
+   Qiskit Nature.
+
+   1. Clone the Qiskit Nature repository.
+
+      .. code:: sh
+
+         git clone https://github.com/Qiskit/qiskit-nature.git
+
+   2. Cloning the repository creates a local folder called ``qiskit-nature``.
+
+      .. code:: sh
+
+         cd qiskit-nature
+
+   3. If you want to run tests or linting checks, install the developer requirements.
+
+      .. code:: sh
+
+         pip install -r requirements-dev.txt
+
+   4. Install ``qiskit-nature``.
+
+      .. code:: sh
+
+         pip install .
+
+   If you want to install it in editable mode, meaning that code changes to the
+   project don't require a reinstall to be applied, you can do this with:
+
+   .. code:: sh
+
+      pip install -e .
+
+
+.. _optional_installs:
+
+Optional installs
+=================
+
+Qiskit Nature supports the use of different classical libraries and programs, via drivers, which
+compute molecular information, such as one and two body integrals. This is needed as problem input
+algorithms that compute properties of molecules, such as ground state energy, so at least one such
+library/program should be installed. As you can choose which driver you use, you can install as
+many, or as few as you wish, that are supported by your platform etc.
+
+See `Driver installation <./apidocs/qiskit_nature.drivers.html#drivers>`__ which lists each driver
+and how to install the dependent library/program that it requires.
+
+----
+
+Ready to get going?...
+======================
+
+.. raw:: html
+
+   <div class="tutorials-callout-container">
+      <div class="row">
+
+.. customcalloutitem::
+   :description: Find out about Qiskit Nature and how to use it for natural science problems.
+   :header: Dive into the tutorials
+   :button_link:  ./tutorials/index.html
+   :button_text: Qiskit Nature tutorials
```

Comment:
Is this the only reason for all those custom templates added by this PR? Rendering the docs locally is really underwhelming to be honest..
I also did not see all of these extensions in Terra. Did you copy them from another Qiskit repo?
![screenshot_1621233675](https://user-images.githubusercontent.com/21973473/118443014-abc02c00-b6eb-11eb-83a6-254fb6f62c44.png)

Reply:
This follows the look/feel of the GettingStarted in the main Qiskit documentation and the extensions are defined in `qiskit` repo. Yes the module is a copy - I need to check with Paul concerning lack of copyright as I copied it as is and it does not have an.